### PR TITLE
Re-organize tests,Add resource/limits/runtimeclass

### PR DESCRIPTION
**Pull request by krsna1729**
Currently cpumanagerpolicy:static is partially broken.
Hope these files catch future regressions

Added a script to generate yamls for different runtimeclass from
a template with different resource constraints on the pod

Signed-off-by: Saikrishna Edupuganti <saikrishna.edupuganti@intel.com>

### CODE REVIEW

**Comment by krsna1729** [comment]
@egernst @jcvenegas

File: clr-k8s-examples/tests/cpumanager/test-cpumanager.yaml.tmpl [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+kind: Pod
+metadata:
+  name: test-cpumanager-burstable-integer-limit-__runtimeclass__
+spec:
+  restartPolicy: Never
+  containers:
+  - name: busy
+    image: busybox
+    command: [ "top" ]
+    resources:
```
Comment (egernst) [inline clr-k8s-examples/tests/cpumanager/test-cpumanager.yaml.tmpl]: Would it be useful to have multi-container pods here?  Can help test placement inside the pod as well as on host.

Comment (egernst) [reply clr-k8s-examples/tests/cpumanager/test-cpumanager.yaml.tmpl]: This becomes relevant when you use Kata...

**Comment by krsna1729** [comment]
@mcastelino fixed.

**Pull request merged**